examples-site/react-components/example.njk

```diff
@@ -0,0 +1,23 @@
+---
+pagination:
+  data: reactComponents.flat
+  size: 1
+  alias: example
+permalink: 'react-components/{{ example.component }}/{{ example.exampleName }}/index.html'
```

Comment:
Maybe it could just be under `react/`? (Might also have things like layouts in future also that aren't components in the general sense.)

```suggestion
permalink: 'react/{{ example.component }}/{{ example.exampleName }}/index.html'
```